apps/api/src/app/auth/auth.controller.ts

```diff
@@ -70,7 +69,7 @@ export class AuthController {
     Logger.verbose('Checking Github Auth');
 
     if (!process.env.GITHUB_OAUTH_CLIENT_ID || !process.env.GITHUB_OAUTH_CLIENT_SECRET) {
-      throw new ApiException(
+      throw new BadRequestException(
```

Comment:
The usage of `BadRequestException` is questionable here, it should be probably `InternalServerErrorException` 

but for now this PR doesn't focus on this, at least now we can see that the exceptions are wrong since they're not being hidden behind `ApiException`